reporter/internal/pdata/generate.go

```diff
@@ -43,6 +46,8 @@ func (p *Pdata) Generate(tree samples.TraceEventsTree,
 	// By specification, the first element should be empty.
 	stringSet.Add("")
 	funcSet.Add(funcInfo{nameIdx: stringSet.Add(""), fileNameIdx: stringSet.Add("")})
+	mappingSet.Add(dummyFileID)
+	dic.MappingTable().AppendEmpty()
```

Comment:
@dmathieu Is there a chance that we can change pprofile.NewProfiles() in a way that the dictionary tables are already set up appropriately with default values?
Because to generate valid Profiles, this is currently done at "thousand" places and instead should be done only at a single place.